automation_tools/__init__.py

```diff
@@ -479,6 +479,16 @@ def setup_foreman_discovery():
     run('rm -rf {0}'.format(template_file))
 
 
+def setup_libvirt_key():
+    """Task to setup key pairs for secure comminication between Satellite server
+    and libvirt hypervisor."""
+    key_url = os.environ.get('LIBVIRT_KEY_URL')
```

Comment:
We do have the corresponding entry updated in jenkins configuration file: `Installation Environment Variables`